src/extension.ts

```diff
@@ -68,8 +69,70 @@ export async function activate(context: vscode.ExtensionContext) {
 			.then(document => vscode.window.showTextDocument(document));
 	};
 
+	const outputChannel = vscode.window.createOutputChannel("Roblox TS");
+
+	const statusBarItem = vscode.window.createStatusBarItem(
+		vscode.StatusBarAlignment.Right,
+		500
+	);
+
+	const statusBarDefaultState = ()=> {
+		statusBarItem.text = "$(debug-start) Start rbx-ts";
+		statusBarItem.command = "roblox-ts.start";
+	}
+
+	let compilerProcess: childProcess.ChildProcessWithoutNullStreams;
+	const startCompiler = async() => {
+		statusBarItem.text = "$(debug-stop) Stop rbx-ts";
+		statusBarItem.command = "roblox-ts.stop";
+
+		if (!vscode.workspace.workspaceFolders) return showErrorMessage("Not in a workspace");
+
+		outputChannel.appendLine("Starting compiler..");
+		outputChannel.show();
+
+		compilerProcess = childProcess.spawn("rbxtsc.cmd", vscode.workspace.getConfiguration("roblox-ts.command").get<Array<string>>("parameters")!, {
```

Comment:
Does `childProcess.spawn("rbxtsc")` not work? `rbxtsc.cmd` isn't cross platform and won't work on MacOS and Linux (WSL).

Reply:
If it doesn't work, we should specify different command strings per platform.

Reply:
Yea, it didn't work for me and returned ENOENT while I tried without .cmd

Reply:
Adding onto this, it may be a better idea to use `npx` so that it'll work on local installs.

Reply:
npx sounds good, I'll implement that - though I don't really have an idea how to do per platform execution

Reply:
Have you tried enabling the shell option? I believe spawn defaults to looking for an executable, rather than running the command in a shell.

Reply:
I have not - I think that will solve it though!

Reply:
Yup, it does solve it

Reply:
Also is `.get<...>("parameters")` guaranteed to exist? Do we need a `?? []` or something here?

Reply:
Also, would be nice to have a "dev" boolean setting to switch to using `rbxtsc-dev` (our developmental version).

Reply:
I think .get should be defined, but there's a second parameter for the default - I'll add that in. A dev boolean should be easy to add too, I'll do that as well